k8s/patches.go

```diff
@@ -160,9 +160,9 @@ func (p *PatchApplier) Apply(resource *unstructured.Unstructured, patchStr strin
 
 	// Finally customize the target resource
 	var out bytes.Buffer
-	if err := kustomize.RunKustomizeBuild(&out, memFS, fakeDir); err != nil {
-		return nil, errors.Wrap(err, "failed to run kustomize build")
-	}
+	//if err := kustomize.RunKustomizeBuild(&out, memFS, fakeDir); err != nil {
+	//return nil, errors.Wrap(err, "failed to run kustomize build")
+	//}
```

Comment:
@moshloop We might need to use https://pkg.go.dev/sigs.k8s.io/kustomize/api/krusty#Kustomizer.Run to replace this